monkey/infection_monkey/utils/random_password_generator.py

```diff
@@ -0,0 +1,6 @@
+import secrets
+
```

Comment:
Create a constant here, called `SECRET_BYTE_LENGTH` and set it to 32. Then use this constant as a default parameter in `get_random_password`. 12 bytes is not enough.